pkg/process/procutil/process_linux_test.go

```diff
@@ -661,8 +661,9 @@ func TestParseStatContent(t *testing.T) {
 	now := time.Now()
 
 	for _, tc := range []struct {
```

Comment:
[extra] Would you mind converting this table tests to use [subtests](https://go.dev/blog/subtests)?